web-animations-1/Overview.bs

```diff
@@ -3201,6 +3201,10 @@ If the [=effect target=] is a [=pseudo-element=], the [=target element=] is
 its [=originating element=] and the [=target pseudo-selector=] is as required
 to specify that particular [=pseudo-element=].
 
+Note that not all [=effect targets=] specified in this manner (such as ''::part()''
+pseudo-elements, unsupported/invalid pseudo-elements, and `null`) have
```

Comment:
I'm not sure invalid pseudo-elements can exist, right? They're rejected at the API boundary?

What does ``null`` refer to here? The lack of a target element? (Represented by `null` in the programming interface) Or the lack of a target pseudo-selector?